shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/java/org/apache/shardingsphere/sql/parser/oracle/visitor/statement/impl/OracleDDLStatementSQLVisitor.java

```diff
@@ -425,6 +425,9 @@ public ASTNode visitAssociateStatistics(final AssociateStatisticsContext ctx) {
             for (IndexNameContext each: ctx.functionAssociation().indexName()) {
                 result.getIndexes().add((IndexSegment) visit(each));
             }
+//            for (FunctionContext each: ctx.functionAssociation().function()) {
+//                result.getFunctions().add((FunctionSegment) visit(each));
+//            }
```

Comment:
Hi @wgy8283335, I tried to parse `function` segment in a similar with as `index` segment. However, with these commented line above I'm getting this error

```
java.lang.ClassCastException: org.apache.shardingsphere.sql.parser.sql.common.value.identifier.IdentifierValue cannot be cast to org.apache.shardingsphere.sql.parser.sql.common.segment.ddl.function.FunctionSegment
```
Do you have idea how to resolve this issue?

Reply:
Maybe could invalidate caches in IntelliJ and maven clean.

Reply:
Thanks for your suggestion. I've tried it, but it gives the same error.

Reply:
Because, you miss a visit function.
![image](https://user-images.githubusercontent.com/22066046/130545185-fa2f9618-f9a9-4245-a3fd-cdf79d34d5b4.png)

Reply:
Oh yes, thank you!!